### PR TITLE
Create Adityakrpand.json

### PR DESCRIPTION
**Pull request by Adityakrpand**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6211"></a>

### CODE REVIEW

File: data/Adityakrpand.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  "bio": "I write code to create solutions that positively impact people's lives"
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/AimlAditya?t=KgtiJz4Wkm9tcB7i3bitrA&s=09",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/adityakrpandey7777/?igshid=MDM4ZDc5MmU%3D,
```
Comment (ChinmayMhatre) [inline data/Adityakrpand.json:13]:
```suggestion
      "url": "https://www.instagram.com/adityakrpandey7777",
```
Missing semi colon and refactored url

File: data/Adityakrpand.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Aditya kr Pandey",
+  "type": "community",
+  "bio": "I write code to create solutions that positively impact people's lives"
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/AimlAditya?t=KgtiJz4Wkm9tcB7i3bitrA&s=09",
```
Comment (ChinmayMhatre) [inline data/Adityakrpand.json:8]:
```suggestion
      "url": "https://twitter.com/AimlAditya",
```

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]
Looks good, mentioned a few changes.

**Review by imbeshat — CHANGES_REQUESTED** [review]

**Comment by Adityakrpand** [comment]
amazing experience

**Review by iamravi1 — APPROVED** [review]
Looks good

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**